packages/drivers/tinylicious-driver/src/insecureTinyliciousUrlResolver.ts

```diff
@@ -12,6 +12,9 @@ import {
 import { ITokenClaims } from "@fluidframework/protocol-definitions";
 import { KJUR as jsrsasign } from "jsrsasign";
 import { v4 as uuid } from "uuid";
+
+const defaultPort = "35843";
```

Comment:
Export this as `tinyliciousDefaultPort` and use in the other `packages/*` packages probably